Enable logging the emerge output as well as any special messages
displayed by the ebuild scripts themselves.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -144,7 +144,7 @@ CXXFLAGS="${CFLAGS}"
 #     If the direcory does not exist, it will be created automatically and
 #     group permissions will be applied to it.  If the directory already
 #     exists, portage will not modify it's permissions.
-#PORT_LOGDIR=""
+PORT_LOGDIR="/var/log/ebuilds"
 #
 # PORTDIR_OVERLAY is a directory where local ebuilds may be stored without
 #     concern that they will be deleted by rsync updates. Default is not
@@ -313,7 +313,7 @@ FEATURES="
 # PORTAGE_ELOG_CLASSES: selects messages to be logged, possible values are:
 #                          info, warn, error, log, qa, *
 #                       Warning: commenting this will disable elog
-#PORTAGE_ELOG_CLASSES="warn error log"
+PORTAGE_ELOG_CLASSES="warn error log"
 
 # PORTAGE_ELOG_SYSTEM: selects the module(s) to process the log messages. Modules
 #                      included in portage are (empty means logging is disabled):
@@ -335,7 +335,7 @@ FEATURES="
 #                      separated list of loglevels to override PORTAGE_ELOG_CLASSES
 #                      for this module (e.g.
 #                        PORTAGE_ELOG_SYSTEM="mail:warn,error syslog:* save")
-#PORTAGE_ELOG_SYSTEM="save mail"
+PORTAGE_ELOG_SYSTEM="save"
 
 # PORTAGE_ELOG_COMMAND: only used with the "custom" logging module. Specifies a command
 #                      to process log messages. Two variables are expanded:
